strongbox-aql/pom.xml

```diff
@@ -157,11 +157,17 @@
             <artifactId>strongbox-storage-api</artifactId>
             <version>${project.version}</version>
         </dependency>
-        
-            
+
+
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <scope>test</scope>
```

Comment:
The scope should probably be moved up to the `strongbox-parent` (here and below).